extensions/ql-vscode/test/vscode-tests/cli-integration/variant-analysis/variant-analysis-manager.test.ts

```diff
@@ -379,7 +380,9 @@ describe("Variant Analysis Manager", () => {
         : "codeql-pack.yml";
       const qlpackContents = load(
         packFS.fileContents(packFileName).toString("utf-8"),
-      );
+      ) as ExtensionPackMetadata & {
+        dependencies: Record<string, string>;
```

Comment:
Should there be a type for the  `{ dependencies: Record<string, string>}` stuff we're doing in a few places? I'm not sure what these are.. is it part of a qlpack file?

Reply:
These are indeed part of the qlpack file, but since they are only used in tests I don't think it's worth it to create a type for it.

Reply:
is https://github.com/github/vscode-codeql/pull/2847/files#diff-ca3de75e87f4315b8c49f5553191e61416837c90e71cd7df1b4598ff2c28306dR545 something different then?

Reply:
I had missed that, thanks for spotting it. I'll introduce a type for the qlpack file.

Reply:
Ah okay thanks, I wasn't sure. I'm happy to not introduce a type too, but I think it would be helpful.